modules/nw-metallb-frr-k8s-configuration-crd.adoc

```diff
@@ -0,0 +1,403 @@
+// Module included in the following assemblies:
+//
+// * networking/metallb/metallb-frr-k8s.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="nw-metallb-frrconfiguration-crd_{context}"]
+= Configuring the FRRConfiguration CRD
+
+The following are reference examples that use the `FRRConfiguration` CR.
+
+[id="nw-metallb-frrconfiguration-crd-routers_{context}"]
+== The routers field
+
+You can use the `routers` field to configure multiple routers, one for each Virtual Routing and Forwarding (VRF).
+For each router, you can define the ASN and a list of BGP neighbors to connect to, as in the following example:
```

Comment:
nit: the ASN must be defined. The neighbors are optional though

Reply:
ok. I rewrote as 
"For each router, you must define the ASN and you can also define a list....."